packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -546,7 +546,9 @@ export function signInActor({ services }: SignInMachineOptions) {
         },
         shouldConfirmSignIn: (_, event): boolean => {
           groupLog('+++shouldConfirmSignIn', 'event', event);
-          return event.data.nextStep?.signInStep === 'CONFIRM_SIGN_IN';
+          return (
+            event.data.nextStep?.signInStep === 'CONFIRM_SIGN_IN_WITH_SMS_CODE'
+          );
         },
```

Comment:
Assuming `shouldConfirmSignIn` covers flows that are not SMS related (TOTP for example)?

Reply:
Maybe we should rename `shouldConfirmSignIn` to `shouldConfirmSignInWithSMS`?